corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yaml

```diff
@@ -149,12 +149,25 @@
   since: '2.14'
 
 - id: use_grid_menus
-  name: Use Menu Grids
-  description: Display root menu as a grid instead of a list.
-  widget: bool
+  name: Modules Menu Display
+  description: Display root menu as a list or grid.
+  values: [false, true]
+  value_names: ['List', 'Grid']
   default: false
   since: "2.24"
   toggle: GRID_MENUS
+  foot_note: "Set Grid Menus as No for this to be active"
```

Comment:
I don't think the text you're displaying as a footnote is really different than help text. I'd just add those explanations to `description` and remove the footnote logic altogether.